services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -866,6 +868,77 @@ class ProtectedPathSpec extends BaseSpecification {
         database.resetCosts()
     }
 
+    def "System doesn't reroute main flow path when protected path is broken and new alt path is available\
```

Comment:
What about 
`System doesn't reroute main flow path when protected path is broken and more preferable alt path is available`
?